packages/react-composites/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from '@playwright/test';
+import { MockCallAdapterState } from '../../../common';
+import { addVideoStream, defaultMockCallAdapterState, defaultMockRemoteParticipant } from '../../call/hermetic/fixture';
+import { IDS } from '../../common/constants';
+import { dataUiId, isTestProfileDesktop, pageClick, stableScreenshot, waitForSelector } from '../../common/utils';
+import { loadCallPage, test } from './fixture';
+
+test.describe('Overflow gallery tests', async () => {
+  test('Overflow gallery should be present when people or chat pane are open', async ({
+    page,
+    serverUrl
+  }, testInfo) => {
+    const initialState = createInitialStateWithManyAudioParticipants();
+    await loadCallPage(page, serverUrl, initialState);
+
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page)).toMatchSnapshot('overflow-gallery-with-many-participants.png');
```

Comment:
should this test maybe be skipped all together if a mobile test? there shouldn't be a overflow gallery on mobile because the side pane is the whole screen on mobile.

Reply:
Skipped!